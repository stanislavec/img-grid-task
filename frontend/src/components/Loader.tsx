import React from 'react';
import scss from './Loader.module.scss';

function Loader() {
  return (
    <div className={scss.row}>
      <div className={scss.loader}></div>
    </div>
  );
}

export default Loader;
