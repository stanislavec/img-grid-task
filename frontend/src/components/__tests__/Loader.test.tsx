import { render } from '@testing-library/react';
import Loader from '../Loader';

describe('Loader', () => {
  it('is defined', () => {
    const elem = render(<Loader />);
    expect(elem.baseElement).toBeDefined();
  });

  it('matches snapshot', () => {
    const elem = render(<Loader />);
    expect(elem.container).toMatchSnapshot();
  });
});
