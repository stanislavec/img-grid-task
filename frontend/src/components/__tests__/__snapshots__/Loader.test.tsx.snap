// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader matches snapshot 1`] = `
<div>
  <div
    class="row"
  >
    <div
      class="loader"
    />
  </div>
</div>
`;
